sql-plugin/src/main/scala/com/nvidia/spark/rapids/profiler.scala

```diff
@@ -50,10 +50,13 @@ object ProfilerOnExecutor extends Logging {
   private var isProfileActive = false
   private var currentContextMethod: Method = null
   private var getContextMethod: Method = null
+  private val stageTaskCount = mutable.HashMap[Int, Int]()
+  private var taskLimit = 0
```

Comment:
This name is a bit too generic.  Is it a task limit across all tasks being profiled?  Turns out no, but from the name one cannot tell.
```suggestion
  private var stageTaskLimit = 0
```